10up/ElasticPress
# Fix Horizontal scrolling in Quick Setup Screen

### Description of the Change
This PR fixes the Horizontal scrolling on the Quick Setup screen



Closes #2855 

### How to test the Change


- Reset the setting by using `wp elasticpress settings-reset`
- Go to Settings page
- Observe that the Horizontal scroll is not appearing anymore

### Changelog Entry

> Fixed - Horizontal scroll in ElasticPress Quick Setup Screen 


### Credits

Props @MARQAS 

### Checklist:
